sklearn/manifold/_t_sne.py

```diff
@@ -604,15 +605,9 @@ class TSNE(BaseEstimator):
     n_iter_ : int
         Number of iterations run.
 
-    Examples
+    See Also
     --------
-
-    >>> import numpy as np
-    >>> from sklearn.manifold import TSNE
-    >>> X = np.array([[0, 0, 0], [0, 1, 1], [1, 0, 1], [1, 1, 1]])
-    >>> X_embedded = TSNE(n_components=2).fit_transform(X)
-    >>> X_embedded.shape
-    (4, 2)
+    LocallyLinearEmbedding, Isomap, MDS
```

Comment:
So we would then want to add a short description in each case (typically the first line of the corresponding docstring)

Also @glemaitre  mentionned that `PCA` might be more appropriate.